.coderabbit.yaml

```diff
@@ -0,0 +1,172 @@
+language: en-US
+tone_instructions: ''
+early_access: false
+enable_free_tier: true
+reviews:
+  profile: chill
+  request_changes_workflow: false
+  high_level_summary: false
+  high_level_summary_placeholder: '@coderabbitai summary'
+  high_level_summary_in_walkthrough: false
+  auto_title_placeholder: '@coderabbitai'
+  auto_title_instructions: ''
+  review_status: false
+  commit_status: true
+  fail_commit_status: false
+  collapse_walkthrough: true
+  changed_files_summary: true
+  sequence_diagrams: true
+  assess_linked_issues: false
+  related_issues: true
+  related_prs: true
+  suggested_labels: false
+  auto_apply_labels: false
+  suggested_reviewers: false
+  auto_assign_reviewers: false
+  poem: true
+  labeling_instructions: []
+  path_filters:
+    - '!docs/**'
+  path_instructions:
+    - path: '**/*.*'
```

Comment:
We can probably limit this to PHP/JS/TS files.

Reply:
Limited to php, js, ts, and tsx.  I think there are rules in here that are applicable to some other file types, but we can iterate on those going forward.